### DISCUSSION TITLE
AWS IAM policy tl;dr

### DISCUSSION THREAD

**Discussion by bryantbiggs** [kedacore/keda-docs#740] [Q&A]
Hi - is there a simple list of what IAM policy permissions are required for keda? I know it will vary based on the scaler used but it would be great to know what permissions are required similar to https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/example-iam-policy.json

If I can figure out what permissions are required (AWS IAM only), I can add something for keda here https://github.com/terraform-aws-modules/terraform-aws-iam/tree/master/modules/iam-role-for-service-accounts-eks

**Comment by zroubalik** [comment]
Unfortunately we don't have any list with this info. But would be definitely great to have it there!
